nixops_gcp/nix/gce.nix

```diff
@@ -445,14 +479,14 @@ in
         )
        (filter (fs: fs.gce != null) (attrValues config.fileSystems))));
 
-    deployment.autoLuks =
-      let
-        f = name: dev: nameValuePair (get_disk_name dev)
-          { device = name;
-            autoFormat = true;
-            inherit (dev) cipher keySize passphrase;
-          };
-      in mapAttrs' f (filterAttrs (name: dev: dev.encrypt) config.deployment.gce.blockDeviceMapping);
+    #deployment.autoLuks =
```

Comment:
what's going on with autoluks ? :) or is it now in a separate plugin ?

Reply:
It is a separate 'module' that is defined here : https://github.com/nix-community/nixos-modules-contrib/blob/master/auto-luks/default.nix

I commented out this temporarily because that would require including the **nixos-modules-contrib** dependency which is not fully ready yet

Reply:
can you create an issue to track that.

Reply:
Fixed per https://github.com/nix-community/nixos-modules-contrib/commit/6e4d21f47f0c40023a56a9861886bde146476198 & https://github.com/nix-community/nixops-gce/pull/9/commits/3ce6e4492140a55b5d22ebb1be26b063afd2055e